src/Altinn.Platform/Altinn.Platform.Authentication/Altinn.Platform.Authentication.Tests/Repositories/OrgLookupTest.cs

```diff
@@ -17,21 +20,35 @@ namespace Altinn.Platform.Authentication.Tests.Repositories
     /// </summary>
     public class OrgLookupTest
     {
+        private readonly IMemoryCache _memoryCache;
+        private readonly HttpClient _client;
+        private readonly Mock<ILogger<OrganisationRepository>> _loggerMock = new Mock<ILogger<OrganisationRepository>>();
+        private readonly Mock<IOptions<GeneralSettings>> _optionsMock = new Mock<IOptions<GeneralSettings>>();
+
+        public OrgLookupTest()
+        {
+            var services = new ServiceCollection();
+            services.AddMemoryCache();
+            var serviceProvider = services.BuildServiceProvider();
+
+            _memoryCache = serviceProvider.GetService<IMemoryCache>();
+            _client = new HttpClient();
```

Comment:
@SandGrainOne not sure if we should mock the messagehandler or if it is okay to simply call the cdn?

Reply:
It should be mocked.